api/src/main/java/org/openmrs/module/queue/utils/PrivilegeConstants.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.queue.utils;
+
+import org.openmrs.annotation.AddOnStartup;
+import org.openmrs.annotation.HasAddOnStartupPrivileges;
+
+/**
+ * Contains all privilege names and their descriptions. Some privilege names may be marked with
+ * AddOnStartup annotation.
+ *
+ * @see org.openmrs.annotation.AddOnStartup
+ * @since 1.8
```

Comment:
https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#JavaConventions-NewPublicMethodsandClasses

Reply:
I have never seen version 1.8 of this module 😊

Reply:
Oh my bad, let me fix that, just to be clear,  is `2.4.0` the current version?!

Reply:
true dat!